website/webpack.config.js

```diff
@@ -88,6 +69,15 @@ module.exports = {
           chunks: [example],
         })
     ),
+    ...examples.map(
+      example =>
+        new CopyWebpackPlugin([
```

Comment:
I don't think we have to call the plugin again, we can provide an array of paths as an argument.

Reply:
I tried that but it was a nightmare. Having a regex in the `from` and using it in the `to` doesn't work as expected.

Reply:
Not sure to follow, we can't do this?

```js
[
  new CopyWebpackPlugin([
    ...examples.map(example => ({
      from: path.join(__dirname, example, "assets"),
      to: path.join(outputPath, example, "assets")
    })),
    {
      from: path.join(__dirname, "assets"),
      to: "assets/"
    },
    path.join(__dirname, "_redirects")
  ])
]
```

Reply:
Ah right! Yes, this works!